packages/core/src/components/section/section.tsx

```diff
@@ -147,9 +154,11 @@ export const Section: React.FC<SectionProps> = React.forwardRef((props, ref) =>
         }
     }, [collapseProps, isCollapsed, isControlled]);
 
-    const isHeaderLeftContainerVisible = title != null || icon != null || subtitle != null;
     const isHeaderRightContainerVisible = rightElement != null || collapsible;
 
+    const sectionId = uniqueId("section");
+    const sectionTitleId = title ? uniqueId("section-title") : undefined;
```

Comment:
```suggestion
    const sectionTitleId = title != null ? uniqueId("section-title") : undefined;
```

Reply:
Line 174 has logic
```javascript
 {title && (
                <div
```
so we would want this logic to be the same.

Reply:
We could change that to `title != null` too though. Depends whether you want empty string to create the title element.

Reply:
ok, good point, let's keep what you have